docs/src/build_tips.md

```diff
@@ -120,6 +120,29 @@ There are plans to handle file changes in the wizard automatically [(#25)](https
 Note that you can include local files like patches very easily by placing them within
 a `bundled/patches` nested directory, and then providing `"./bundled"` as one of the `sources` for your build.  See, for example, [`OpenBLASBuilder`](https://github.com/JuliaLinearAlgebra/OpenBLASBuilder/blob/5a6eca317de505abc3678e47f08f4355646f511e/build_tarballs.jl#L9).
 
+## Using GCC on macOS and FreeBSD
+
+For these target systems Clang is the default compiler, however some programs may not be compatible with Clang.
+
+For programs built with CMake (see the [CMake build](#CMake-builds-1) section) you can use the GCC toolchain file that is in `/opt/${target}/${target}_gcc.toolchain`.
+
+If the project that you want to build uses the GNU Build System (also knows as the Autotools), there isn't an automatic switch to use GCC, but if have to set the appropriate variables.  For example, this setting can be used to build most C/C++/Fortran programs with GCC for FreeBSD and macOS:
+```sh
+if [[ "${target}" == *-freebsd* ]] || [[ "${target}" == *-apple-* ]]; then
+    CC=/opt/${target}/bin/${target}-gcc
+    CXX=/opt/${target}/bin/${target}-g++
+    FC=/opt/${target}/bin/${target}-gfortran
+    LD=/opt/${target}/bin/${target}-ld
+    AR=/opt/${target}/bin/${target}-ar
+    AS=/opt/${target}/bin/${target}-as
+    NM=/opt/${target}/bin/${target}-nm
+    OBJDUMP=/opt/${target}/bin/${target}-objdump
```

Comment:
Now that we have JIT compiler wrappers, we can just tell people to use `${target}-gcc` or even `gcc` if they want.  :)

Reply:
So I guess we only need to set
```
CC=gcc
CXX=g++
```
and stop?